libiocage/lib/Prompts.py

```diff
@@ -17,10 +18,19 @@ def release(self):
                 print(f"[{i}] {available_release.name}")
             i += 1
 
-        default_release = available_releases[default]
-        selection = input(f"Release ({default_release.name}) [{default}]: ")
+        if default is not None:
+            default_release = available_releases[default]
+            selection = input(f"Release ({default_release.name}) [{default}]: ")
+        else:
+            default_release = None
+            selection = input("Your selection: ")
 
         if selection == "":
+            if default_release is None:
+                raise libiocage.lib.errors.DefaultReleaseNotFound(
+                    host_release_name=self.host.release_version,
```

Comment:
Where is this defined? I don't see it, or `self.host.distribution.releasesc` (above) in `__init__`